src/javascripts/app.js

```diff
@@ -6,9 +6,10 @@ import routes from './routes'
 import { AppContainer } from 'react-hot-loader'
 import { Router, browserHistory } from 'react-router'
 
-// Do anything on startup here
+window.onload = function() {
+    document.getElementById("message-input").focus();
+};
```

Comment:
This would be better in your `ChatInput` component `componentDidMount`, using a `ref` to get the input